script/release.sh

```diff
@@ -71,6 +71,7 @@ clone() {
   cd $CLI
   git remote remove origin
   git remote add origin git@github.com:$ORIGIN_REPO/$CLI.git
+  git remote add upstream git@github.com:$UPSTREAM_REPO/$CLI.git
```

Comment:
I'd prefer it SSH to be honest... Having to enter http credentials each-time is a kill-joy. I have no arguments to say that ssh is any better haha. Just personal preference of mine.